app/connectors/MiddleConnector.scala

```diff
@@ -52,6 +52,9 @@ trait MiddleConnector {
   def connectToAts(UTR: SaUtr, taxYear: Int)(implicit hc: HeaderCarrier): Future[AtsData] =
     http.GET[AtsData](url("/taxs/" + UTR + "/" + taxYear + "/ats-data"))
 
+  def connectToPayeAts(nino: Nino, taxYear: Int)(implicit hc: HeaderCarrier): Future[AtsData] =
```

Comment:
Missing tests

Reply:
Test added. Please see https://github.com/hmrc/tax-summaries-frontend/pull/72 to see where this is called.